app/uk/gov/hmrc/gform/gform/handlers/DeclarationControllerRequestHandler.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.gform.handlers
+
+import play.api.libs.json.JsValue
+import uk.gov.hmrc.gform.sharedmodel.form.{ Form, FormField }
+
+import scala.util.Try
+
+class DeclarationControllerRequestHandler extends FormDataUpdater {
+
+  def handleUpdatedFormRequest(requestBody: Option[JsValue], form: Form): Option[Form] =
+    for {
+      body  <- requestBody
+      field <- Try(body.as[FormField]).toOption
```

Comment:
There is `asOpt` method, no need for `Try`

Reply:
Oh Cool! I didn't like it as well